js/language_server/Makefile

```diff
@@ -2,9 +2,19 @@
 default: build
 
 include ../engine/Makefile.include
+include ../tree-sitter/Makefile.include
+
+# TODO: manage this better
+LANGUAGES := bash c cairo cpp csharp dart dockerfile elixir go hack html java jsonnet kotlin lisp lua ocaml php promql protobuf python r rust solidity terraform typescript vue
+TS_LANGS := bash c cairo clojure cpp c_sharp dart dockerfile elixir go hack html java jsonnet kotlin lua ocaml php promql proto python r rust solidity hcl typescript vue
```

Comment:
could use a comment explaining what the differences between `LANGUAGES` and `TS_LANGS` is